### DISCUSSION TITLE
Audio Delay

### DISCUSSION THREAD

**Discussion by Normann-git** [iina/iina#5003] [Ideas]
Regarding audio delay settings there are 0.5s steps available often not being suitable because they're much too large. Would appreciate having smaller steps like 0,1s in both directions!

Would it be possible to add it in the future?

**Comment by low-batt** [comment]
I assume the request is for a setting that could cause the `Audio Delay` menu items and the `Audio delay` slider on the `AUDIO` tab of the quick settings panel to be more fine grained? So that if you used the keys bound to the menu item the incremental change would be smaller? That sounds like a useful feature.

On the quick settings panel you can specify a specific delay as shown below. Of course not as convenient as being able to incrementally change the delay.
![delay](https://github.com/iina/iina/assets/86170219/735057ac-5a33-47af-890f-cf405bb7761e)

**Reply by Normann-git** [reply]
It's quite tricky to fill in that 0,1 or any other value on quick settings because it will always reset to 0,5 by each additional adjustment. So you have to start all over again. But I have to agree- it's better than having the 0,5 or even no audio delay control at all.

I really hate VLC-Player but appropriate settings over there work surprisingly better. My personal production chain generates an audio lag not exceeding 0,1 (100 ms), so that's why I would love to have the 0,1 steps with IINA, at least being achieved by quick settings or even better by using the shortcuts.

**Comment by low-batt** [comment]
We don't keep track of discussions, so I have entered a feature request for this in issue #5010. Have a look at that issue and please correct me if I got something wrong.

I also found that by default IINA has keys that will adjust the delay by 0.1s. See that issue for details.

**Comment by low-batt** [comment]
And of course it is possible to change key bindings and assign keys to change the delay by a different amount.
